apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/service/appdata/TrimService.java

```diff
@@ -226,19 +226,18 @@ private int doTrimDerivedTablesOnHeight(int height, boolean isSharding) {
 
         for (DerivedTableInterface table : dbTablesRegistry.getDerivedTables()) {
             log.trace("Try to acquire lock...");
-            globalSync.readLock();
-            log.trace("Got it.");
+//            globalSync.readLock();
             try {
                 long startTime = System.currentTimeMillis();
                 table.prune(pruningTime);
                 table.trim(height, isSharding);
                 dataSource.commit(false);
                 long duration = System.currentTimeMillis() - startTime;
                 // do not log trim duration here, instead go to the logback config and enable trace logs for BasicDbTable class
-                //                    log.trace("Trim of {} took {} ms", table.getName(), duration);
+//                log.debug("Trim of {} took {} ms", table.getName(), duration);
```

Comment:
Maybe set it to trace back? Too many debug output will be present in the log now

Reply:
That is commented out, so I don't think that is important here.
Usually it's faster to uncomment logging here then configure TRACE logging in XML.